<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessForm</class>
 <widget class="QWidget" name="ChessForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>736</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/rook_ico.png</normaloff>:/res/rook_ico.png</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>3</y>
     <width>712</width>
     <height>719</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="boardWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>570</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="navigateWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="leftBtn">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="startBtn">
             <property name="toolTip">
              <string notr="true">定位至棋局开始</string>
             </property>
             <property name="statusTip">
              <string notr="true"/>
             </property>
             <property name="whatsThis">
              <string notr="true"/>
             </property>
             <property name="accessibleName">
              <string notr="true"/>
             </property>
             <property name="accessibleDescription">
              <string notr="true"/>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/left1.svg</normaloff>:/res/left1.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="backBtn">
             <property name="toolTip">
              <string notr="true"> 后退 一步</string>
             </property>
             <property name="statusTip">
              <string notr="true"/>
             </property>
             <property name="whatsThis">
              <string notr="true"/>
             </property>
             <property name="accessibleName">
              <string notr="true"/>
             </property>
             <property name="accessibleDescription">
              <string notr="true"/>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/left.svg</normaloff>:/res/left.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="goBtn">
             <property name="toolTip">
              <string notr="true"> 前进 一步</string>
             </property>
             <property name="statusTip">
              <string notr="true"/>
             </property>
             <property name="whatsThis">
              <string notr="true"/>
             </property>
             <property name="accessibleName">
              <string notr="true"/>
             </property>
             <property name="accessibleDescription">
              <string notr="true"/>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/right.svg</normaloff>:/res/right.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="otherBtn">
             <property name="toolTip">
              <string notr="true"> 变着</string>
             </property>
             <property name="statusTip">
              <string notr="true"/>
             </property>
             <property name="whatsThis">
              <string notr="true"/>
             </property>
             <property name="accessibleName">
              <string notr="true"/>
             </property>
             <property name="accessibleDescription">
              <string notr="true"/>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/more.svg</normaloff>:/res/more.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="endBtn">
             <property name="toolTip">
              <string notr="true">定位至棋局结尾</string>
             </property>
             <property name="statusTip">
              <string notr="true"/>
             </property>
             <property name="whatsThis">
              <string notr="true"/>
             </property>
             <property name="accessibleName">
              <string notr="true"/>
             </property>
             <property name="accessibleDescription">
              <string notr="true"/>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/right1.svg</normaloff>:/res/right1.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="rightBtn">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="infoTabWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="moveTab">
         <attribute name="title">
          <string notr="true"> 着法</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="infoTab">
         <attribute name="title">
          <string notr="true"> 信息</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="studyTabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="noteTab">
       <attribute name="title">
        <string> 笔记</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="previerTab">
       <attribute name="title">
        <string> 预览</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
